### DISCUSSION TITLE
Net::SSH::AuthenticationFailed):

### DISCUSSION THREAD

**Discussion by aselims** [basecamp/kamal#679] [Q&A]
$ Kamal setup shows this
INFO [81198188] Running /usr/bin/env mkdir -p .kamal on 1x.x.134
  Finished all in 0.4 seconds
  ERROR (Net::SSH::AuthenticationFailed): Exception while executing on host 1x.x.134: Authentication failed for user root@1x.x.134

How I can fix this issue or specify which certificate to use? I am using the alias Kamal docker.
P.S. I can access the server with my ssh key from terminal/ vscode

**Comment by acidtib** [comment]
Hi @aselims this solutions from @fennekit may be of help to you 

https://github.com/basecamp/kamal/issues/427#issuecomment-1889768865

**Reply by abratashov** [reply]
also, you can upload your ssh key https://github.com/basecamp/kamal/issues/427#issuecomment-1968994170
